docs/performance_metrics.md

```diff
@@ -27,3 +29,81 @@ To generate metrics data for selected performance tests, e.g. boot time only:
+|            | boot_time_pmem_ms                          | 2              | 10             |
+|            | boot_time_16_vcpus_ms                      | 2              | 10             |
+|            | boot_time_16_vcpus_pmem_ms                 | 2              | 10             |
+| Virtio Net | virtio_net_latency_us                      | 10             | 5              |
+|            | virtio_net_throughput_single_queue_rx_gbps | 10             | 5              |
+|            | virtio_net_throughput_single_queue_tx_gbps | 10             | 5              |
+|            | virtio_net_throughput_multi_queue_rx_gbps  | 10             | 5              |
+|            | virtio_net_throughput_multi_queue_tx_gbps  | 10             | 5              |
+| Block      | block_read_MiBps                           | 10             | 5              | 
+|            | block_write_MiBps                          | 10             | 5              |
+|            | block_random_read_MiBps                    | 10             | 5              |
+|            | block_random_write_MiBps                   | 10             | 5              |
+|            | block_multi_queue_read_MiBps               | 10             | 5              |
+|            | block_multi_queue_write_MiBps              | 10             | 5              |
+|            | block_multi_queue_random_read_MiBps        | 10             | 5              |
+|            | block_multi_queue_random_write_MiBps       | 10             | 5              |
+|            | block_read_IOPS                            | 10             | 5              |
+|            | block_write_IOPS                           | 10             | 5              |
+|            | block_random_read_IOPS                     | 10             | 5              |
+|            | block_random_write_IOPS                    | 10             | 5              |
+|            | block_multi_queue_read_IOPS                | 10             | 5              |
+|            | block_multi_queue_write_IOPS               | 10             | 5              |
+|            | block_multi_queue_random_read_IOPS         | 10             | 5              |
+|            | block_multi_queue_random_write_IOPS        | 10             | 5              |
+
+## Output Format
+
+Performance-metrics output the result into a json file if `report-file` param is set.
+The fields included in JSON include:
+
+| Field Name         | Content                                  |
+|--------------------|------------------------------------------|
+| git_human_readable | Recent tag information of git repository |
+| git_revision       | Commit id of HEAD                        |
+| git_commit_date    | Commit date of HEAD                      |
+| date               | Date for executing the program           |
+| results            | A list of metrics                        |
+
+Such as:
```

Comment:
Better say:  "A sample example is below."

Reply:
Fixed.